pkg/deployment/deployment.go

```diff
@@ -315,6 +315,32 @@ func (builder *Builder) WithServiceAccountName(serviceAccountName string) *Build
 	return builder
 }
 
+// WithVolumes sets list of volumes that can be mounted by containers belonging to the deployment.
+func (builder *Builder) WithVolumes(deployVolumes []coreV1.Volume) *Builder {
```

Comment:
Can you please change it to WithVolume and accept a single coreV1.Volume? Multiple volumes can be chained together

Reply:
thanks, updated